Mathlib/MeasureTheory/Function/SimpleFunc.lean

```diff
@@ -1156,8 +1156,8 @@ open Finset Function
 theorem support_eq [MeasurableSpace α] [Zero β] (f : α →ₛ β) :
     support f = ⋃ y ∈ f.range.filter fun y => y ≠ 0, f ⁻¹' {y} :=
   Set.ext fun x => by
-    simp only [mem_support, Set.mem_preimage, mem_filter, mem_range_self, true_and_iff, exists_prop,
-      mem_iUnion, Set.mem_range, mem_singleton_iff, exists_eq_right']
+    simp only [mem_support, Set.mem_preimage, Finset.mem_filter, mem_range_self, true_and_iff,
```

Comment:
I know what I did! I accidentally created a `MeasureTheory.Finset` namespace. Then any `open Finset` written inside the `MeasureTheory` namespace would open that new one instead of `_root_.Finset`.